src/neuroconv/datainterfaces/ecephys/baserecordingextractorinterface.py

```diff
@@ -72,6 +71,15 @@ def get_metadata(self):
 
         return metadata
 
+    def synchronize(self, starting_time: Optional[float] = None, timestamps: Optional[ArrayType] = None):
+        if starting_time is not None or timestamps is not None:
+            assert starting_time != timestamps, "Specify either 'starting_time' or 'timestamps', but not both!"
+
+        if starting_time is not None:
+            self.recording_extractor.set_times(self.recording_extractor.frame_to_times() + starting_time)
+        elif timestamps is not None:
+            self.recording_extractor.set_times(timestamps)
```

Comment:
There's really no point in defining that abstract method if you're not even going to call it in child classes